src/openms/include/OpenMS/FORMAT/MzMLFile.h

```diff
@@ -175,16 +175,25 @@ namespace OpenMS
     */
     bool isSemanticallyValid(const String& filename, StringList& errors, StringList& warnings);
 
+
+    struct SpecInfo
+    {
+      int count_centroided = 0;
```

Comment:
reason why it is an  int?

Reply:
usually the fastest type on a platform. nothing else.
Or would you rather prefer `UInt`?

Reply:
it counts so I would say size_t